vuejs/resources/web-types/vue@2.7.0.web-types.json

```diff
@@ -0,0 +1,994 @@
+            },
+            {
+              "name": "camel",
+              "description": "Transform the kebab-case attribute name into camelCase."
+            },
+            {
+              "name": "sync",
+              "description": "A syntax sugar that expands into a `v-on` handler for updating the bound value"
+            }
+          ]
+        },
+        {
+          "name": "model",
+          "description": "Create a two-way binding on a form input element or a component. For detailed usage and other notes, see the Guide section linked below.",
+          "doc-url": "https://vuejs.org/v2/api/#v-model",
+          "attribute-value": {
+            "type": "any",
+            "required": true
+          },
+          "argument": {
+            "name": "v-model prop/event pair to bind",
+            "pattern": {
+              "items": {
+                "path": "attributes",
+                "filter": "match-with-target-list",
+                "target-list": "/js/events",
+                "prefix": "update:"
+              }
+            }
+          },
+          "modifiers": [
+            {
+              "name": "lazy",
+              "description": "Listen to `change` events instead of `input`"
+            },
+            {
+              "name": "number",
+              "description": "Cast valid input string to numbers"
+            },
+            {
+              "name": "trim",
+              "description": "Trim input"
+            }
+          ]
+        },
+        {
+          "name": "pre",
+          "description": "Skip compilation for this element and all its children. You can use this for displaying raw mustache tags. Skipping large numbers of nodes with no directives on them can also speed up compilation.",
+          "doc-url": "https://vuejs.org/v2/api/#v-pre",
+          "attribute-value": {
+            "kind": "no-value"
+          }
+        },
+        {
+          "name": "cloak",
+          "description": "This directive will remain on the element until the associated Vue instance finishes compilation. Combined with CSS rules such as `[v-cloak] { display: none }`, this directive can be used to hide un-compiled mustache bindings until the Vue instance is ready.",
+          "doc-url": "https://vuejs.org/v2/api/#v-cloak",
+          "attribute-value": {
+            "kind": "no-value"
+          }
+        },
+        {
+          "name": "once",
+          "description": "Render the element and component *once* only. On subsequent re-renders, the element/component and all its children will be treated as static content and skipped. This can be used to optimize update performance.",
+          "doc-url": "https://vuejs.org/v2/api/#v-once",
+          "attribute-value": {
+            "kind": "no-value"
+          }
+        }
+      ]
+    },
+    "css": {
+      "pseudo-elements": [
+        {
+          "name": "v-deep",
```

Comment:
::v-deep This is now also deprecated in vue@2.7. Like in was in vue@3.2,    also :deep(child-selector) should be added

Warning is: `[@vue/compiler-sfc] ::v-deep usage as a combinator has been deprecated. Use :deep(<inner-selector>) instead.`

Reply:
![image](https://user-images.githubusercontent.com/136992/177212340-8414e11f-998b-425f-b6c9-fe9da3e41cc5.png). WebStorm (Latest EAP). Shows `unknown pseudo element dee` warning.

Reply:
I didn't find anything about deep in the vue 2 official documents or the changelog.
Instead, I think it's not a feature of vue but the vue-loader, as it's described in https://vue-loader.vuejs.org/guide/scoped-css.html#deep-selectors.

Reply:
Yes, it's from the loader, but that's also the case with `v-deep`. I changed my code from v-deep to deep, and it behaves the same.

Reply:
The vue@3.2 web types already have deep declared.

Reply:
https://github.com/vuejs/rfcs/blob/master/active-rfcs/0023-scoped-styles-changes.md RFC for that.